docs/source/configuration/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: Apollo Router instruments
+title: GraphOS Router Instruments
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)